src/mixins/test/wordWrap.test.js

```diff
@@ -0,0 +1,14 @@
+import wordWrap from '../wordWrap'
+
+describe('wordWrap', function() {
+
+  it('should accept other values', function() {
+    expect(wordWrap('break-all')).toMatchSnapshot()
+  })
+
+  it('should default wrap to break-word', function() {
+    expect(wordWrap()).toMatchSnapshot()
+  })
```

Comment:
Since we are going to be porting this over as "live" examples we should set up tests similar to this:

```javascript
expect({
  div: wordWrap()
}).toMatchSnapshot()
```